thunder/torch/__init__.py

```diff
@@ -670,6 +670,11 @@ def expand(a: TensorLike, /, *shape: int) -> TensorLike:
     return clang.expand(a, *shape)
 
 
+@torchsymbol(torch.Tensor.expand_as, is_method=True)
+def expand_as(a: TensorLike, /, b: TensorLike) -> TensorLike:
```

Comment:
fyi @IvanYashchuk and @nikitaved; we may want to change functions like `expand_as` and `full_like` to not be symbols in the future. I filed https://github.com/Lightning-AI/lightning-thunder/issues/362 to track this

@k223kim This is exactly what we should do for now, except let's change the signature to be:

```
def expand_as(a: TensorLike, b: TensorLike, /) -> TensorLike:
```